terraform/sqs.tf.json.template.py

```diff
@@ -29,18 +33,21 @@
                                 "maxReceiveCount": 9 if retry else 1,
                                 "deadLetterTargetArn": "${aws_sqs_queue.%s.arn}"
                                                        % config.unqual_tallies_queue_name(retry=not retry, fail=retry)
-                            })
+                            }),
+                            "tags": get_cloud_tags(config.tallies_queue_name(retry=retry)),
                         }
                         for retry in (False, True)
                     },
                     config.unqual_notifications_queue_name(fail=True): {
                         "name": config.notifications_queue_name(fail=True),
                         "message_retention_seconds": 14 * 24 * 60 * 60,
+                        "tags": get_cloud_tags(config.qualified_resource_name('fail')),
                     },
                     config.unqual_tallies_queue_name(fail=True): {
                         "fifo_queue": True,
                         "name": config.tallies_queue_name(fail=True),
                         "message_retention_seconds": 14 * 24 * 60 * 60,
+                        "tags": get_cloud_tags(config.qualified_resource_name('fail', suffix='.fifo')),
```

Comment:
This looks wrong. It's out of sync with the actual name. This code recently changed so this is probably a merge error.

Anyways, we can already see how brittle this duplication is. Ideally we would want to automate it.

```suggestion
                        "tags": get_cloud_tags("${self.name}"),
```

This doesn't currently work with TF but let's pretend that it does and resolve `${self.name}` in a post-processing step.

Let's also have `${self.name}` be the default such that we can call it as in

```suggestion
                        "tags": get_cloud_tags(),
```